server/src/main/java/com/epam/aidial/core/server/controller/AppSchemasController.java

```diff
@@ -0,0 +1,124 @@
+package com.epam.aidial.core.server.controller;
+
+import com.epam.aidial.core.config.Config;
+import com.epam.aidial.core.metaschemas.MetaSchemaHolder;
+import com.epam.aidial.core.server.ProxyContext;
+import com.epam.aidial.core.server.util.ProxyUtil;
+import com.epam.aidial.core.storage.http.HttpStatus;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.vertx.core.Future;
+import io.vertx.core.http.HttpServerRequest;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class AppSchemasController implements Controller {
```

Comment:
Please use singular: AppSchemaController. To be more like others

Reply:
aplied